plugins/tiddlywiki/tiddlyweb/sse-client.js

```diff
@@ -0,0 +1,41 @@
+/*\
+title: $:/plugins/tiddlywiki/tiddlyweb/sse-client.js
+type: application/javascript
+module-type: startup
+
+GET /recipes/default/tiddlers/:title
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "/events/plugins/tiddlywiki/tiddlyweb";
+exports.after = ["startup"];
+exports.synchronous = true;
+exports.platforms = ["browser"];
+exports.startup = function () {
+  //make sure we're actually being used
+  if($tw.syncadaptor.name !== "tiddlyweb") return;
```

Comment:
Could we instead install the event handler when we startup the syncadaptor?

Reply:
I'm sure we could but I don't really know where. I did it here because that is right after the syncer and sync adapter are initialized. But you could just as easily do it there.